Heuristic.java

```diff
@@ -3,10 +3,9 @@
 	static int[] weights;
 	static int weightCounter;
 
-	public static int evaluate(State s) {
+	public static int evaluate(int[][] field, int rowsCleared) {
```

Comment:
How come you wanna change the input parameters? I was thinking can just make a copy of the state, run the move we wanna test, and pass it into the heuristic function. Then the original `PlayerSkeleton.java` will not have such a huge mess. Like we can abstract out this mess into the `Heuristic.java class`.

Reply:
I think I understand what you are looking for. I can transfer the codes into the heuristic.java